client/state/sites/test/selectors.js

```diff
@@ -36,29 +39,37 @@ describe( 'selectors', () => {
 			expect( site ).to.be.null;
 		} );
 
-		it( 'should return the site object', () => {
+		it( 'should return a normalized site with computed attributes', () => {
 			const site = getSite( {
 				sites: {
 					items: {
-						2916284: { ID: 2916284, name: 'WordPress.com Example Blog', URL: 'https://example.com' }
+						2916284: {
+							ID: 2916284,
+							name: 'WordPress.com Example Blog',
+							URL: 'https://example.com',
+							options: {
+								unmapped_url: 'https://example.wordpress.com'
+							}
+						}
 					}
 				}
 			}, 2916284 );
 
-			expect( site ).to.contain( { ID: 2916284, name: 'WordPress.com Example Blog' } );
-		} );
-
-		it( 'should return computed attributes', () => {
-			const site = getSite( {
-				sites: {
-					items: {
-						2916284: { ID: 2916284, name: 'WordPress.com Example Blog', URL: 'https://example.com' }
-					}
+			expect( site ).to.eql( {
```

Comment:
It's odd that this passes locally, but fails on CircleCI. Is `chai` comparing the reference of options, not deeply? Are we running into memoise cache problems?